pkg/controller/certificates/trigger/trigger_controller.go

```diff
@@ -159,7 +159,7 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 		now := c.clock.Now()
 		retryAfter := crt.Status.LastFailureTime.Add(retryAfterLastFailure)
 		if now.Before(retryAfter) {
-			log.Info("Not re-issuing certificate as an attempt has been made in the last hour", "retry_after", retryAfter)
+			log.V(logf.DebugLevel).Info("Not re-issuing certificate as an attempt has been made in the last hour", "retry_after", retryAfter)
```

Comment:
This sounds like something that would be useful at Info level, if cert-manager is in production and admins are trying to figure out why a cert isn't being issued.